src/locator/index.ts

```diff
@@ -212,6 +212,20 @@ function scorePattern(point: Point, ratios: number[], matrix: BitMatrix) {
   }
 }
 
+function recenterLocation(matrix: BitMatrix, p: Point): Point {
```

Comment:
Any reason we wouldn't recenter vertically you?

Reply:
I didn't think it would matter, but after testing, it appears that also leads to a slight improvement.

a52cda5